authentik/stages/authenticator_webauthn/settings.py

```diff
@@ -0,0 +1,13 @@
+"""Stage authenticator webauthn Settings"""
+
+from celery.schedules import crontab
+
+from authentik.lib.utils.time import fqdn_rand
+
+CELERY_BEAT_SCHEDULE = {
+    "stages_authenticator_webauthn_import_mds": {
+        "task": "authentik.stages.authenticator_webauth.tasks.webauthn_mds_import",
+        "schedule": crontab(minute=fqdn_rand("webauthn_mds_import"), hour="*/4"),
```

Comment:
I think we could probably do even less frequently as the files only change with a new authentik build

Reply:
Yeah I just picked another settings.py to copy from. What would you suggest?

Reply:
Every 24 hours? Could probably even do every week since even upon upgrade there should rarely be important things in here 


Actually that reminds me, we do the check for overdue system tasks on startup, would that also trigger this task on a first startup? If not we should probably change the date set in the prefill decorator to also run those tasks on first startup

Reply:
> would that also trigger this task on a first startup

From my understanding of the code it should. I'll try it out nonetheless.

Reply:
> > would that also trigger this task on a first startup
> 
> I'll try it out nonetheless.

Yeah it does, so we just need to increase the time to like 1 week or something.